packages/platform/src/lib/nx-plugin/executors/build/executor.ts

```diff
@@ -0,0 +1,8 @@
+import { BuildExecutorSchema } from './schema';
+
+export default async function runExecutor(options: BuildExecutorSchema) {
```

Comment:
suggestion: let's just remove executors if we're not having any executors

Reply:
Agreed! 

My initial idea was to have analog executors for serve, build, serve-production-files that would take care of most of the paths configuration internally, and also allow us to support e2e tests based on a node server using the build output.

The way I understand [this issue](https://github.com/nrwl/nx/issues/10814) is that the executor would need to return the host and port to make the cypress executor work with it. But this could be a future PR, too. 

I would remove the executors for now!